doc/userGuide.md

```diff
@@ -338,6 +338,18 @@ To exclude the test case with variation ```-Xmx1024m```:
 
 ```auto exclude test jdk_test_1```
 
+To exclude the test for openj9 only:
+
+```auto exclude test jdk_test impl=openj9```
+
+To exclude the test for java 8 only:
+
+```auto exclude test jdk_test ver=8```
```

Comment:
We are using ver to describe subset, is the plan to eventually match it to what it is named in the playlist (as impl to impl)?  It will be easier for people to remember usage.

Reply:
Yes, the plan to change subset to ver in the future.